projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.ts

```diff
@@ -10,17 +10,23 @@ import {
   Self,
   SimpleChanges,
   ViewChild,
+  ViewEncapsulation,
 } from '@angular/core';
 import { TSChartSeries, TSChartSettings } from './model/ts-chart-settings';
 import { UplotSyncService } from './uplot-sync-service';
 import { UPlotUtils } from '../uplot/uPlot.utils';
+import { tooltipPlugin } from './tooltip-plugin';
+import { ExecutionTabContext } from '../execution-page/execution-tab-context';
+import { TSTimeRange } from './model/ts-time-range';
+import { AlignedData } from 'uplot';
 
 declare const uPlot: any;
```

Comment:
I've tried to play with it to avoid any, and found the following example:
https://stackblitz.com/edit/angular-ivy-k6ig7h?file=src%2Fapp%2Fapp.component.ts

In step, I was able to include it in the following way
```
import { AlignedData, Options, Cursor } from 'uplot'; // required uplot types
//@ts-ignore
import uPlot = require('uplot'); // uplot class itself
```
Yes we use `@ts-ingore` to write such import, but it forces the IDE to check uPlot types more strict and as a result write better code. `declate const uPlot: any` wont' require any more

Interesting moment, I've checked the assembled scripts and didn't notice that this line add any real import. So we still need to include uPlot sources in angular.json scripts sections.
But in the example from the stackblitz, it looks like it work as a real import. That's the point for further investigations

Reply:
We should do the change now @lucianbaciu and create a ticket for solving the import vs. angular.js issue. (Would be great not to have uplot defined in there)

Reply:
will fix this!